packages/graphql/src/mutations/_txHelper.js

```diff
@@ -141,6 +167,22 @@ export default function txHelper({
     const shouldUseRelayer = useRelayer({ mutation, value })
     const shouldUseProxy = await useProxy({ proxy, addr, to, from, mutation })
     debug(`shouldUseProxy: ${shouldUseProxy}`)
+    debug(`shouldUseRelayer: ${shouldUseRelayer}`)
+
+    // Get our gas price
+    if (!shouldUseRelayer && ['mainnet', 'rinkeby'].includes(contracts.net)) {
+      if (window.ethereum) {
+        // Don't use eth_gasPrice for MM
+        try {
+          gasPrice = await fetchGasPrice()
+        } catch (err) {
+          debug(`Error fetching gas price`, err)
```

Comment:
Should we log to Sentry ?

Reply:
I don't think there's been any Sentry integration in graphql(yet).

Reply:
Ah yes, very good point. You are still very lucid at this late hour you know... :)